cmake/kobj.cmake

```diff
@@ -22,6 +22,7 @@ function(gen_kobj gen_dir_out)
     --kobj-otype-output ${KOBJ_OTYPE}
     --kobj-size-output ${KOBJ_SIZE}
     $<$<BOOL:${CMAKE_VERBOSE_MAKEFILE}>:--verbose>
+    DEPENDS $ENV{ZEPHYR_BASE}/scripts/gen_kobject_list.py
```

Comment:
Should use ${ZEPHYR_BASE} instead (for portability reasons).

Reply:
That was what was originally there but it failed one of the automated tests.

Reply:
Then there is a bug elsewhere, ${ZEPHYR_BASE} must always be used, or else one risks hiding a portability bug.

Reply:
Ah, well, too late now.